docs/developer/advanced/upgrading-nodejs.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[upgrading-nodejs]]
+== Upgrading Node.js
+
+{kib} requires a specific Node.js version to run.
+When running {kib} from source, the developer needs to have this version installed locally.
+
+The required version of Node.js is listed in several different files throughout the {kib} source code.
+All of these needs to be updated when upgrading Node.js:
+
+  - {kib-repo}blob/{branch}/.ci/Dockerfile[`.ci/Dockerfile`] - The version is specified in the `NODE_VERSION` constant.
+  - {kib-repo}blob/{branch}/.node-version[`.node-version`]
+  - {kib-repo}blob/{branch}/.nvmrc[`.nvmrc`]
+  - {kib-repo}blob/{branch}/package.json[`package.json`] - The version is specified in the `engines.node` field.
+
+See PR {kib-repo}pull/86593[#86593] for an example of how the Node.js version has been upgraded previously.
+
+In the `6.8` branch, the `.ci/Dockerfile` file does not exist, so when upgrading Node.js in that branch, just skip that file.
```

Comment:
Version numbers should be in regular font.

Reply:
Fixed 👍